lvmd/device_class_manager.go

```diff
@@ -11,7 +11,13 @@ import (
 // ErrNotFound is returned when a VG or LV is not found.
 var ErrNotFound = errors.New("device-class not found")
 
-const defaultSpareGB = 10
+type LVType string
```

Comment:
This type is used to specify the type of VG or pool, it is not for LV, I guess. Is `DeviceType` better?

Reply:
ack, sounds good.